src/Sulu/Bundle/AdminBundle/Tests/Unit/Admin/View/FormOverlayListViewBuilderTest.php

```diff
@@ -287,11 +287,11 @@ public function testBuildFormWithRouterAttributesToFormRequest(): void
             ->setFormKey('role_details')
             ->addListAdapters(['tree'])
             ->addRouterAttributesToFormRequest(['webspace' => 'webspaceId', 'parent' => 'parentId'])
-            ->addRouterAttributesToFormRequest(['locale'])
+            ->addRouterAttributesToFormRequest(['locale' => 'en_US'])
```

Comment:
this example is misleading the logic in JavaScript is something like this:

```
$routerAttributesToFormRequest = ['locale'];

$attributes = [];
foreach ($routerAttributesToFormRequest as $source => $target) {
    if (is_numeric($source)) {
         $source = $target;
    }
    
    $attributes[$target] = $routerAttributes[$source];
}
```

So this code and the other `en_US` changes should be reverted as it is missleading:

```suggestion
            ->addRouterAttributesToFormRequest(['locale'])
```